.github/workflows/build_hub.yml

```diff
@@ -78,40 +77,43 @@ jobs:
           files: prime-router/build/test-results/testIntegration/**/*.xml
           check_name: "Integration Test Results"
 
-  smoke_tests:
-    name: Smoke Tests
-    needs: pre_job
-    if: ${{ needs.pre_job.outputs.has_router_change == 'true' }}
-    runs-on: ubuntu-latest
-    defaults:
-      run:
-        working-directory: prime-router
+      - name: Spin up vault
+        timeout-minutes: 1
+        run: |
+          mkdir .vault/env
+          touch .vault/env/.env.local
+          docker-compose -f docker-compose.yml up -d vault
+          while [ ! -s .vault/env/.env.local ]; do sleep 1; done
+          echo "loaded env vars"
 
-    steps:
-      - name: "Check out changes"
-        uses: actions/checkout@v2
+      - name: Make ftps dir
+        run: mkdir build/ftps
 
-      # Appears not to be needed on GitHub (but needed when running act [https://github.com/nektos/act] locally)
-      # - name: Install docker-compose
-      #   run: apt-get update && apt-get --yes install docker-compose
+      - name: Spin up containers
+        run: |
+          docker-compose -f docker-compose.yml up -d prime_dev redox ftps sftp azurite
 
-      - name: Build Prime Router Package
-        run: bash ./cleanslate.sh
-
-      - name: Run Smoke Tests
-        run: bash ./build.sh -- gradle testSmoke
-      
-      - name: Dump docker logs on failure
-        if: failure()
-        uses: jwalton/gh-docker-logs@v2.0.2
-        with:
-          images: 'prime-router_settings,prime-router_prime_dev'
+      - name: Load settings
+        run: docker-compose -f docker-compose.yml up settings
 
+      - name: Correct sftp perms
+        run: docker-compose exec -T sftp chmod 777 /home/foo/upload
```

Comment:
Interesting.  You had to do this for Github?

Reply:
yeah, the perms aren't translating when being mounted in the docker container.